docs/manual/common/concepts/ComponentTechnologies.md

```diff
@@ -0,0 +1,21 @@
+# Component technologies
+
+As a complete microservices platform, Lagom assembles a collection of technologies and adds value on top of them. Some of the libraries, tools, and servers that Lagom uses and supports were developed at [Lightbend](https://lightbend.com), others are third-party and open-source. As you develop with Lagom, you can also take advantage of these technologies:
+
+* Akka --- Lagom [[Persistence|PersistentEntity]], [[Publish-Subscribe|PubSub]], and [[Cluster|Cluster]] are implemented on top of [Akka](http://akka.io/), Lightbend's toolkit for building concurrent, distributed, and resilient message-driven applications. This is an implementation detail that does not directly concern you when developing simple microservices. However, advanced users can call some Akka APIs directly. Specifically:
```

Comment:
Added link, removed "some" and "advanced". Akka Persistence is mentioned, in the first sentence.